src/aoc2020/day1.clj

```diff
@@ -0,0 +1,143 @@
+(ns aoc2020.day1
+  (:require [aoc2020.util :refer [file->seq]]))
+
+
+;; part 1
+
+(defn strs->ints [strs]
```

Comment:
Doesn't follow naming conventions.

Reply:
If you intend to create fns with seq args that are just a mapping over those args... make sure you have the isolated (testable) variant as well

Reply:
> If you intend to create fns with seq args that are just a mapping over those args... make sure you have the isolated (testable) variant as well

you mean I need to create testable cases(small size test cases) in the Rich Comment Block to test the function?

Reply:
> Doesn't follow naming conventions.

Here is what I read from the [clojure style guide](https://github.com/bbatsov/clojure-style-guide):
<img width="476" alt="Screenshot 2023-05-02 at 6 11 18 PM" src="https://user-images.githubusercontent.com/132013808/235797026-efe70f1c-9f31-4633-9149-f36e499e23f7.png">

This function is converting string sequence to integer sequence, so I used `->`. You mean it should not be a conversion name or the `->` is right but `strs` and `ints` are not suitable?

Reply:
> > Doesn't follow naming conventions.
> 
> This function is converting string sequence to integer sequence, so I used `->`. You mean it should not be a conversion name(like `parse-sequence`) or the `->` is right but `strs` and `ints` are not suitable?

This is attempting to work on entire sequences. Aside from the fact that it shouldn't, the naming convention for seq parameters is `-s`

Reply:
> > If you intend to create fns with seq args that are just a mapping over those args... make sure you have the isolated (testable) variant as well
> 
> You mean I need some test cases in the Rich Comment Block for this function?

1 - I am referring to the functionality as applied to scalars
2 - You should have demonstration code in the RCB for _EVERY FUNCTION_. A large part of the reason you are running into these problems is b/c you are viewing this from the programming side of the equation rather than the usage side.